src/widgets/dnshole/DnsHoleControls.tsx

```diff
@@ -201,38 +253,51 @@ function DnsHoleControlsWidgetTile({ widget }: DnsHoleControlsWidgetProps) {
                 >
                   <Image src={app.appearance.iconUrl} width={40} height={40} fit="contain" />
                 </Box>
-                <Stack spacing="0rem">
+                <Stack spacing="0rem" align="center">
```

Comment:
Having the text in the center of the elements under but not as a whole, and those elements under not being centered either, it kinda looks strange to me.
That is totally up for debate, no idea what the others think.

I suggest, either don't align, or add `style={{flex: 1}}`.

>old
>![image](https://github.com/ajnart/homarr/assets/26098587/7de8e0b1-5926-4226-9987-9bf6ff02cf20)

>new
>![image](https://github.com/ajnart/homarr/assets/26098587/d3f6b397-8d45-4083-9254-5d4384b4a1c7)

> with flex: 1
> ![image](https://github.com/ajnart/homarr/assets/26098587/89de1e46-5fae-4381-bbeb-26fc38013949)